.gitignore

```diff
@@ -7,3 +7,4 @@ DerivedData/
 db.sqlite
 .swiftpm
 .env
```

Comment:
Add `.env.*` while you're at it

Reply:
I think that makes sense only if we add `! .env.example`